test/helpers/newRegistry.js

```diff
@@ -0,0 +1,34 @@
+import config from '../../conf/config.json'
+const paramConfig = config.paramDefaults;
+
+export default async (registryName, token, registryFactory, challengeFactory) => {
+  const registryReceipt = await registryFactory.newRegistryBYOToken(
+    token.address,
+    [
+      paramConfig.minDeposit,
```

Comment:
not sure if this is used anywhere, but this `minDeposit` is not set in Wei

Reply:
tried setting it in wei and it breaks the script, not sure why. should probably leave it as is for now and get this merged before fixing

Reply:
ok...I usually write TODO comments for that. but as long as you know to investigate later